### PR TITLE
Add tinygltf

### PR DESCRIPTION
**Pull request by headshog**

### CODE REVIEW

File: projects/tinygltf/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    # sydr-fuzz -c loader_example.toml cov-export -- -format=lcov > loader_example.lcov
+    # genhtml -o loader_example-html loader_example.lcov
+
+Check security predicates:
+
+    # sydr-fuzz -c loader_example.toml security
+
+Supported fuzz targets:
+
+    # loader_example
```
Comment (kobrineli) [inline projects/tinygltf/README.md:46]: better use `*` or `-` here before target names to create enumeration

File: projects/tinygltf/build.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+################################################################################
+
+export MAIN_OBJ=""
+
+if [[ $CONFIG = "libfuzzer" ]]
+then
+    export CC="clang"
+    export CXX="clang++"
+    export CFLAGS="-g -O0 -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero"
+    export CXXFLAGS="-g -std=c++11 -O0 -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero"
```
Comment (kobrineli) [inline projects/tinygltf/build.sh:25]: do we need to compile with `-O0` optimization level? here and later

File: projects/tinygltf/payload.gltf
Comment (kobrineli) [inline projects/tinygltf/payload.gltf:1]: do we need this file here?

File: projects/tinygltf/loader_example.cc
Comment (kobrineli) [inline projects/tinygltf/loader_example.cc:1]: remove

**Review by kobrineli — CHANGES_REQUESTED** [review]

File: projects/tinygltf/loader_example.cc
Comment (headshog) [reply projects/tinygltf/loader_example.cc:1]: don't need to remove

File: projects/tinygltf/build.sh (hunk 2 above)
Comment (headshog) [reply projects/tinygltf/build.sh:25]: this option was in project's makefile build

File: projects/tinygltf/README.md (hunk 1 above)
Comment (headshog) [reply projects/tinygltf/README.md:46]: done

File: projects/tinygltf/payload.gltf
Comment (headshog) [reply projects/tinygltf/payload.gltf:1]: removed

File: projects/tinygltf/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM sydr/ubuntu20.04-sydr-fuzz
+
+RUN git clone https://github.com/syoyo/tinygltf.git && cd tinygltf && \
+    git checkout 0fa56e239c77cc864dc248842e8887d985cf8e3f
```
[the remark this answers is not in the document]
Comment (headshog) [reply projects/tinygltf/Dockerfile]: done

**Pull request merged**